### PR TITLE
zh-cn: init the translation of "Understanding WCAG"

### PR DESCRIPTION
**Pull request by yin1999**
### Description

init the translation of "Understanding WCAG"

Translation reference resources:

- https://www.w3.org/Translations/WCAG21-zh/
- https://baike.baidu.com/item/%E6%89%A7%E4%B8%9A%E5%BE%8B%E5%B8%88/4625421

### CODE REVIEW

File: files/zh-cn/web/accessibility/understanding_wcag/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+slug: Web/Accessibility/Understanding_WCAG
+l10n:
+  sourceCommit: d52001165c52cca8a43c9bf2f021ac0d498fccb6
+---
+
+{{QuicklinksWithSubPages}}
+
+这一组文章提供了帮助你理解如何遵循 W3C Web 内容无障碍指南 2.0 或 2.1（或者只是 WCAG，出于撰写本文的目的）建议的步骤的快速解释。
+
+WCAG 2.0 和 2.1 为使得 Web 内容更易于让各中残障人士访问提供了一套详细的指南。它非常全面，但也过于详细，因此很难快速理解。因此，我们总结了满足不同建议所需的实际步骤，并在必要时提供了更多详细信息的链接。
```
Comment (yin1999) [inline files/zh-cn/web/accessibility/understanding_wcag/index.md:12]:
```suggestion
WCAG 2.0 和 2.1 为使得 Web 内容更易于让各类残障人士访问提供了一套详细的指南。它非常全面，但也过于详细，因此很难快速理解。为此，我们总结了满足不同建议所需的实际步骤，并在必要时提供了指向更多详细信息的链接。
```

File: files/zh-cn/web/accessibility/understanding_wcag/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## 四大原则
+
+WCAG 大致分为四个原则——它们是 Web 内容无障碍所**必须具备**的主要要素（请参阅[理解无障碍的四大原则](https://www.w3.org/TR/UNDERSTANDING-WCAG20/intro.html#introduction-fourprincs-head)以了解 WCAG 的定义）。
+
+下面的每个链接都将带你进入进一步扩展这些领域的页面，为你提供有关如何编写 Web 内容以符合（进一步细分了原则的）WCAG 2.0 和 2.1 指南中概述的成功标准的实际建议。
+
+- [可感知](/zh-CN/docs/Web/Accessibility/Understanding_WCAG/Perceivable)：用户必须能够通过一种或多种感官的方式来感知内容。
+- [可操作](/zh-CN/docs/Web/Accessibility/Understanding_WCAG/Operable)：用户必须能够控制 UI 元素（例如，按钮必须可以通过某种方式（鼠标、键盘、语音指令等）点击）。
+- [可理解](/zh-CN/docs/Web/Accessibility/Understanding_WCAG/Understandable)：内容必须是能够被用户理解的。
+- [健壮性](/zh-CN/docs/Web/Accessibility/Understanding_WCAG/Robust)：内容必须使用广泛采用的，且将能够在不同的浏览器中持续工作的 Web 标准来开发。
```
Comment (yin1999) [inline files/zh-cn/web/accessibility/understanding_wcag/index.md:23]:
```suggestion
- [健壮性](/zh-CN/docs/Web/Accessibility/Understanding_WCAG/Robust)：内容必须使用广泛采用的，并且能够在不同的浏览器中长期可用的 Web 标准来开发。
```

File: files/zh-cn/web/accessibility/understanding_wcag/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+title: 理解 Web 内容无障碍指南
+slug: Web/Accessibility/Understanding_WCAG
+l10n:
+  sourceCommit: d52001165c52cca8a43c9bf2f021ac0d498fccb6
+---
+
+{{QuicklinksWithSubPages}}
```
Comment (jasonren0403) [inline files/zh-cn/web/accessibility/understanding_wcag/index.md:8]:
```suggestion
{{AccessibilitySidebar}}
```

File: files/zh-cn/web/accessibility/understanding_wcag/index.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+---
+title: 理解 Web 内容无障碍指南
+slug: Web/Accessibility/Understanding_WCAG
+l10n:
+  sourceCommit: d52001165c52cca8a43c9bf2f021ac0d498fccb6
```
Comment (jasonren0403) [inline files/zh-cn/web/accessibility/understanding_wcag/index.md:5]:
```suggestion
  sourceCommit: 4e689448e34b2a4bfaa3a8462903cde4ab53643e
```

**Review by jasonren0403 — COMMENTED** [review]
reflect recent sidebar-related changes

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**